packages/forma-36-website/src/pages/components/modal/index.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: 'Modal'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Modal'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-modal--default'
+---
+
+Modal dialogs interrupt users and demand an action. They are appropriate when user’s attention needs to be directed toward important information.
```

Comment:
Modal dialogs interrupt a user by isolating information that needs to be acted upon. They are often triggered by an action a user took, and should be used to inform, confirm or complete a task.